core/src/main/scala/org/bitcoins/core/dlc/accounting/RateOfReturnUtil.scala

```diff
@@ -2,7 +2,9 @@ package org.bitcoins.core.dlc.accounting
 
 object RateOfReturnUtil {
 
+  /** @see https://alvinalexander.com/scala/how-to-format-numbers-commas-international-currency-in-scala/ */
   def prettyPrint(ror: BigDecimal): String = {
-    (ror * 100).toString() + "%"
+    val percent = ror * 100
+    f"${percent}%1.2f" + "%"
```

Comment:
is it possible to have it only display the decimal place when it's not zero, that way it is `100%` instead of `100.00%`?

Reply:
I can't find how to do this easily. It is probably possible with java libraries, but I'm a little worried about introducing java specific libraries into `core` as they may not be supported in scalajs land.